packages/stratocacher-layer-simple-object/src/index.js

```diff
@@ -5,29 +5,43 @@ import Q from "q";
 // Be careful with this!
 //  * It caches by _reference_ (by default)!
 //  * In _never_ evicts!
-const CACHE = {};
+
+let CACHE = {};
+function cache({cid}) {
+	if (!CACHE[cid]) {
+		CACHE[cid] = {};
+	}
+	return CACHE[cid];
+}
+
```

Comment:
This isn't really related to adding the DynamoDB layer. Please open a separate PR for this.

Reply:
ok